centreon/www/api/class/centreon_ceip.class.php

```diff
@@ -124,6 +125,20 @@ private function getVisitorInformation(): array
             ? 'admin'
             : 'user';
 
+        $countAcl = (int) $this->sqlFetchValue(
+            <<<SQL
+            SELECT COUNT(*) AS countAcl
+            FROM acl_actions_rules AS aar
```

Comment:
Why you use acl_actions_rules table while we need topology and where is the relation with contact_id or contact groups of the user?